qemu-user-blacklist.txt

```diff
@@ -20,6 +20,7 @@ dbus
 diffutils
 digikam
 dovecot
+dtk6core
```

Comment:
I guess it should be after `dqlite` to keep alphabetical order

Reply:
Fixed.